website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -4607,14 +4610,16 @@ window.
 
 Beam provides one data-driven trigger,
 <span class="language-java">`AfterPane.elementCountAtLeast()`</span>
-<span class="language-py">`AfterCount`</span>. This trigger works on an element
+<span class="language-py">`AfterCount`</span>
+<span class="language-go">`Trigger{Kind: ElementCountTrigger, ElementCount: 2}`</span>. This trigger works on an element
 count; it fires after the current pane has collected at least *N* elements. This
 allows a window to emit early results (before all the data has accumulated),
 which can be particularly useful if you are using a single global window.
 
 It is important to note that if, for example, you specify
 <span class="language-java">`.elementCountAtLeast(50)`</span>
-<span class="language-py">AfterCount(50)</span> and only 32 elements arrive,
+<span class="language-py">AfterCount(50)</span>
+<span class="language-go">`Trigger{Kind: ElementCountTrigger, ElementCount: 50}`</span> and only 32 elements arrive,
```

Comment:
For contrast, this one indicates a specific number.

Note how verbose this is compared to Java and Python. Perhaps we should have helper functions like `window.TriggerAfterCount(50)` to make the user experience user easier? (it avoids users accidentally constructing bad triggers too)